ros2bag/ros2bag/verb/burst.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from argparse import FileType
+
+from rclpy.qos import InvalidQoSProfileException
+from ros2bag.api import check_not_negative_int
+from ros2bag.api import check_path_exists
+from ros2bag.api import check_positive_float
+from ros2bag.api import convert_yaml_to_qos_profile
+from ros2bag.api import print_error
+from ros2bag.verb import VerbExtension
+from ros2cli.node import NODE_NAME_PREFIX
+from rosbag2_py import get_registered_readers
+from rosbag2_py import Player
+from rosbag2_py import PlayOptions
+from rosbag2_py import StorageOptions
+import yaml
+
+class BurstVerb(VerbExtension):
+    """Burst data from a bag."""
+
+    def add_arguments(self, parser, cli_name):  # noqa: D102
+        reader_choices = get_registered_readers()
```

Comment:
Non-blocking - mildly wondering if this could be refactored out into common code, as a lot of it looks like copy-paste from `play.py`

Reply:
I agree that it could, but I think I'd prefer to do that in a separate PR.